src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateEditModelValidator.cs

```diff
@@ -31,11 +30,13 @@ public ProviderApplicationsReportCreateEditModelValidator(ITimeProvider timeProv
                 RuleFor(x => x.ToDate)
                     .Cascade(CascadeMode.StopOnFirstFailure)
                     .Must(date => date.AsDateTimeUk() < timeProvider.NextDay)
-                    .WithMessage("Date to cannot be in the future");
+                    .WithMessage("Date to cannot be in the future")
+                    .Must((model, x) => model.ToDate.AsDateTimeUk() < model.FromDate.AsDateTimeUk().Value.AddMonths(3))
```

Comment:
`AddMonths` does exist, "it's surprising what you didn't know until .................."